src/dataviews/histogram-dataview/histogram-data-model.js

```diff
@@ -1,4 +1,5 @@
 var _ = require('underscore');
+var BackboneCancelSync = require('../../util/backbone-abort-sync');
```

Comment:
Minor thing... we call this `BackboneAbortSync` [here](https://github.com/CartoDB/cartodb.js/blob/v4/src/dataviews/category-dataview/search-model.js#L3)...